lib/index.js

```diff
@@ -175,7 +175,7 @@ module.exports = function loader (css, map) {
          * @param {String} css  Result (Raw Module)
          * @param {Object} map  Source Map
          */
-        return cb(null, css, map)
+        return cb(null, css, map) || null
```

Comment:
```js
cb(null, css, map)
return null
```

Reply:
I'm working on a React app at Homeaway that spits out a long string of broken promise warnings every time I build, and either of these two fixes makes them all go away.
Seems like a trivial fix to a constantly annoying problem. 
Please merge one or the other. Thank you.

Reply:
@DmitriWolf Are you also using `bluebird` ?

Reply:
We are!